### DISCUSSION TITLE
Build error due to a fail download of "ivy-publish" plugin

### DISCUSSION THREAD

**Discussion by christianCasRo** [osmandapp/OsmAnd#18309] [General]
First of all, good day
In my project  that I was using lately  since the Friday I have an error when I tried to build and run the app,
The Android Studio don’t move from "Gradle project sync in process..."
The notification became on loops trying to download the "xml" files 
![image](https://github.com/osmandapp/OsmAnd/assets/119369433/bde49e33-98c6-42a8-a6c9-68141edf2aba)

Then in the build you can check that the URL failed to load: https://builder.osmand.net/ivy/...
![image](https://github.com/osmandapp/OsmAnd/assets/119369433/bebcbafb-9e67-4476-89fc-0bf2b92e5e65)

maybe is a temporal error from the server but I haven´t find any news about from  any osmand social networks
If you know anything about pls tell me, 

Thank in advance

PS: I think if you have it in cache you don´t need to download, Because I have a lot of copies from my project but I have that problem once, then I thought that it was a my own problem from Android Studio so I deleted the cache of Studio and I have this problem in every project
now.

**Comment by vshcherb** [comment]
Correct url is https://creator.osmand.net/ivy/

**Reply by christianCasRo** [reply]
Yes thanks you, 
I think I used an old one, but it was working untill that moment.
Just a couples of restart and working !
